Resources/ServerInfo/Guidebook/_CD/sop/core.xml

```diff
@@ -0,0 +1,61 @@
+<Document>
+  # GENERAL STANDARD OPERATING PROCEDURES
+
+  The standard procedures followed on any uniform NT station. The Captain can edit and make additions to their station's general operation procedures (excluding Contraband) whenever they please as long as a copy of the changes is forwarded to each station bound fax machine. Anyone in the station's sector is required to follow this document and their related work SOP.
+
+  ## Secured Rooms
+
+  Used when determining secure trespass from regular trespass. This is a list of most if not every room that is considered secure. For edge cases the Head of Security and Captain can agree to retroactively declare a room secure.
+
+  - AI core and control rooms.
+  - Armory.
+  - Gravity generator room.
+  - Head of staff's personal rooms.
+  - Main atmospherics room.
+  - Server rooms, including camera, crew monitoring, research, and telecommunications.
+  - Station wide power generation room.
+  - Vault.
```

Comment:
Should bridge be a secure area?

Reply:
Hm- It's a toss up question. Because they can't really _do_ much without access, but it's still a place with a ton of secure terminals.

Reply:
Intuitively, I would say it is. However, I do think that it is no more risky than the security break room.